utils/fetchLsp8Metadata.ts

```diff
@@ -188,5 +181,13 @@ const getLsp8Metadata = async (
     })
   })
 
-  return validateLsp4MetaData(lsp8Metadata)
+  console.debug('LSP8 metadata', {
```

Comment:
Remove this?

Reply:
I would like to keep debug logs at least for some time as it help while investigating issues.